tests/genomics/test_mibig_loader.py

```diff
@@ -79,4 +79,4 @@ def test_parse_bgc_metadata_json():
     bgc = parse_bgc_metadata_json(str(json_file))
     assert isinstance(bgc, BGC)
     assert bgc.bgc_id == "BGC0000001"
-    assert bgc.mibig_bgc_class == ["Polyketide"]
+    assert bgc.mibig_bgc_class == ("Polyketide",)
```

Comment:
Why do we need the comma here?

Reply:
`("sth")` is same as as string `"sth"`. Giving a comma, you will get a tuple not string.